app/views/users/show.html.erb

```diff
@@ -15,13 +15,24 @@
     <span><%=@user.bio%></span>
   </div>
  <div class="posts">
+    <%@user.most_recent_post.each do |post|%>
       <div class="post-info">
-        <span>Post Title:<%=@posts.title%></span>
-        <span><%=@posts.text%></span><br>
-        <span>Comments:<%=@posts.comments_counter%> Likes:<%=@posts.likes_counter%></span>
+        <span>Post Title:<%=post.title%></span>
+        <span><%=post.text%></span><br>
+        <span>Comments:<%=post.comments_counter%> Likes:<%=post.likes_counter%></span>
       </div>
+    <%end%>
   </div>
   <%= link_to "/users/#{@user.id}/posts" do %>
     <button type="button" class="all-posts">See all posts</button>
   <% end %>
+
+  <div>
+    <h1>Add a New Post</h1>
+    <%= form_with model:@post, url: user_posts_path(:author_id) do |form|%>
+      <%= form.text_field :title, placeholder: "Add Title"%>
+      <%= form.text_field :text, placeholder: "Add Text"%>
+      <%=form.submit%>
+    <%end%>
```

Comment:
- [ ] After working out the errors, I thought I could add a post, but the form is not working or giving any error(s). Kindly fix that.
- [ ] When users click on a link of a user on the home page, there is an error saying that there is no user with `id=1`, this is because the user show action is not properly done. Kindly fix these errors and submit them for another review.

![usersshow](https://user-images.githubusercontent.com/53124548/219845955-ec358587-1cd2-4e14-8e1f-e96f4b3fb8ab.png)

![indexuser](https://user-images.githubusercontent.com/53124548/219845978-d7f96c22-7084-482f-87c3-36bb393e387c.png)